src/components/v2/devtronStackManager/DevtronStackManager.tsx

```diff
@@ -190,6 +190,11 @@ export default function DevtronStackManager({
                         ...currentModule,
                         installationStatus: result.status,
                     })
+                    if(moduleInInstallingState === result.name && result.status !== ModuleStatus.INSTALLING){
```

Comment:
Repeating code line. Can we optimize this function

Reply:
Done